pyresample/geometry.py

```diff
@@ -298,8 +305,29 @@ def get_bbox_lonlats(self) -> tuple:
                            (s4_lon.squeeze(), s4_lat.squeeze())])
         if hasattr(lons[0], 'compute') and da is not None:
             lons, lats = da.compute(lons, lats)
+        if force_clockwise and not self._corner_is_clockwise(
+                lons[0][-2], lats[0][-2], lons[0][-1], lats[0][-1], lons[1][1], lats[1][1]):
+            # going counter-clockwise
+            # swap the side order and the order of the values in each side
+            # to make it clockwise
+            lons = [lon[::-1] for lon in lons[::-1]]
+            lats = [lat[::-1] for lat in lats[::-1]]
```

Comment:
Should this be it's own function to clarify the code and not need the comments above?

Reply:
Any good name suggestions for the function?

Reply:
`reverse_boundaries` ?